### DISCUSSION TITLE
Some top scores from Stable did not import to Lazer

### DISCUSSION THREAD

**Discussion by iiScrabble** [ppy/osu#25884] [Q&A] [answered]
Hello, I recently installed Lazer for the first time (on Dec. 7th, ver 2023.1130.0) and noticed that a couple of my top performance plays did not import from my stable to Lazer profile. I realize it may be a bit late now for the logs to be useful, but my thought process was to just give it some time to see if the issue resolved itself.

It seems similar to #25397 but I wanted to be certain it's due to the same root cause. The scores in question below were set in stable months prior to downloading Lazer, and the local scores and replays are present in Lazer.

**Stable Profile**
https://osu.ppy.sh/users/3773018

**Stable scores that are missing in Lazer highlighted below**
![Screenshot 2023-12-18 202133](https://github.com/ppy/osu/assets/154292262/2ef2c2ad-9d49-4875-a84e-0b906bc81805)

**Lazer Profile**
https://lazer.ppy.sh/users/3773018

**Logs**

[compressed-logs.zip](https://github.com/ppy/osu/files/13710384/compressed-logs.zip)

**Comment by peppy** [comment, marked as answer]
We'll be doing a complete re-import later this month so please wait for that. It should hopefully fix any edge cases like this.
